x/blocksdk/types/msgs.go

```diff
@@ -0,0 +1,71 @@
+package types
+
```

Comment:
nit: test-cases?

Reply:
Or do we test these cases in the `KeeperTestSuite`

Reply:
We do test these in `KeeperTestSuite` but I think we should have `types` level tests too